### ISSUE TITLE
what is data augmentation trick you added?

### ISSUE THREAD

**Issue by dogyoonlee** [yanx27/Pointnet_Pointnet2_pytorch#69]
what is data augmentation trick you added?
I want to know the classification  accuracy without the additional augmentation trick.
what is the classification accuracy with pure augmentations which are mentioned in pointnet++ official paper?

**Comment by ashlet** [comment]
it seems to be adding normal vx,vy,vz to the points' x,y,z data

**Comment by sheshap** [comment]
@dogyoonlee and @ashlet It seems like they are not doing rotation and jitter but only doing random point drop out, scaling, and shifting.

**Comment by mmmmimic** [comment]
> @dogyoonlee and @ashlet It seems like they are not doing rotation and jitter but only doing random point drop out, scaling, and shifting.

I agree with you and do you think this affects the result to a considerable degree? By the way, I'm confused that how can we define the training set, validation set, and test set in this implementation. Is validation set the test set?
